.github/workflows/build.yml

```
@@ -105,21 +105,20 @@ jobs:
         TF_VAR_paas_app_docker_image: ${{env.DOCKER_REPO}}:${{env.SHA_TAG}}
         ARM_ACCESS_KEY: ${{ secrets.ARM_ACCESS_KEY }}
 
-
-    - name: Terraform plan
-      run: terraform plan -var-file=terraform/workspace-variables/qa.tfvars terraform
+    - name: Terraform Plan and Apply
+      run: |
+        terraform plan -var-file=terraform/workspace-variables/qa.tfvars terraform
+        terraform apply -var-file=terraform/workspace-variables/qa.tfvars -auto-approve terraform
```

```suggestion
        terraform apply -var-file=terraform/workspace-variables/qa.tfvars -auto-approve terraform
```

There seems to be very little point with plan

---

plan shows you the changes that will be applied. it's good to keep running it for troubleshooting